actions/action-lint/README.md

```diff
@@ -0,0 +1,46 @@
+# helm-lint
+
+This action will lint all actions and workflows of a repository.
+
+## Prerequisites
+
+You need to create the lint matcher file `actionlint-matcher.json` and configure it to your liking.
```

Comment:
Would it be possible to checkout/download the default `actionlint-matcher.json` file automatically, so that we don't have to create it in every repo where we want to use this?

Reply:
If we do this there should still be an option to overwrite it in case you need to change it

Reply:
what's the use case for changing it? afaict we would never need that

Reply:
No, we never changed it. I just like the idea of keeping configuration options open. You are right, we can also do it automatically first and change it in the future if the need arises

Reply:
it is not really a configuration file. It only describes how GitHub should parse the output of actionlint to create inline code reviews on PRs